### PR TITLE
added accessedservice_microservices.txt

### PR DESCRIPTION
**Pull request by a20fanma**

### CODE REVIEW

File: backend-models/microservices/accessedservice_microservices.txt
Comment (c21axepe) [inline backend-models/microservices/accessedservice_microservices.txt:1]: Looks pretty good but where is row 152-219?

Comment (c21axepe) [inline backend-models/microservices/accessedservice_microservices.txt:1]: Some deskriptions are vague like the addClass description

**Review by c21axepe — CHANGES_REQUESTED** [review]
Minor changes are requested. Everything else looks good!

**Comment by a20fanma** [comment]
I have made some changes

**Review by c21axepe — APPROVED** [review]
Everything looks good, Ready for merging!

**Comment by a19hakja** [comment]
Merging =)

**Pull request merged**